pkg/kubelet/util/pluginwatcher/plugin_watcher_test.go

```diff
@@ -189,6 +186,9 @@ func TestPluginRegistrationAtKubeletStart(t *testing.T) {
 		}(plugins[i])
 	}
 
+	w := newWatcherWithHandler(t, hdlr, false /* testDeprecatedDir */)
+	defer func() { require.NoError(t, w.Stop()) }()
+
```

Comment:
Since we're processing serially now, there're would be a lot of contention [here](https://github.com/bertinatto/kubernetes/blob/832822b41808f54f563fa513dbb3371164803e44/pkg/kubelet/util/pluginwatcher/example_plugin.go#L103). To avoid that, this starts the channel-reading goroutines earlier.